\section{Abstract}

The Longest Common Subsequence (LCS) problem is common to
bioinformatics and a classic in computer science, generally considered
NP-hard.  The problem is commonly encountered in bioinformatics in the
comparison of DNA which results in strings that are a combination of
DNA base pairs representing an organism's DNA sequence. These
sequences are typically very long in length resulting in a
computationally intensive problem. By determining the longest common
subsequence of DNA, biologists can determine the relative closeness of
two organisms. The goal of this project was to research methods by
which one could parallelize the LCS length calculation problem to
speed up string comparison. Currently, the most common methods used in
bioinformatics are the BLAST and FASTA hueristics. Both algorithms
emphasize speed of comparison at the expense of sensitivity of
determining optimal alignments of sequences. Algorithms like
Smith-Waterman provide exactness at the expense of speed. This project
aims to inspect and evaluate means of calculating the length of the
longest common subsequence, a value useful for calculating the
similarity in two strings, where the longest common subsequence string
is not needed.

